.github/workflows/pr.yaml

```diff
@@ -39,6 +41,18 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-tests.yaml@python-3.12
     with:
       build_type: pull-request
+  conda-java-build:
+    needs: checks
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-java-build.yaml@branch-24.10
```

Comment:
Since we aren't building a conda package of this, we do not need a build job. The `conda-java-build` can be deleted, and leave only `conda-java-tests` below.

The test job will:
- Generate a conda environment with all build dependencies
- Install libkvikio
- Run maven to build the Java bindings and tests
- Run the tests